composer.json

```diff
@@ -37,9 +37,7 @@
     }
   },
   "require": {
-    "php": ">=5.6",
```

Comment:
Make sure that we have the OK from Matthew before removing 5.6 (we have to make sure that no GD client is still using it)